src/iterator.cr

```diff
@@ -460,6 +501,13 @@ module Iterator(T)
     end
   end
 
+  # Returns an iterator that throws away the first *n* elements and only returns
```

Comment:
I'm always not sure with method descriptions if it is better to repeat the method name or use some other word (e.g. don't explain a word with the word itself). So I tried to avoid it - but I think it does make sense to use the word itself as a method is not the same as the language.